### DISCUSSION TITLE
[Question] Lumen 7

### DISCUSSION THREAD

**Discussion by eleimt** [jenssegers/laravel-mongodb#2085] [General] [answered]
Hello.
Is there support for Lumen 7? Since I have an error "unsupported driver [mongodb]."

**Comment by divine** [comment]
Hello,

What version are you trying to install? Please try alpha version.

Thanks.

**Comment by eleimt** [comment]
@divine 
Hey
I am use "jenssegers/mongodb": "dev-develop".

**Reply by divine** [reply]
Hello,

Can you try latest version?

Thanks!

**Reply by eleimt** [reply]
Hello.
Okay, I try latest version on this week.
Thanks!

**Comment by divine** [comment, marked as answer]
Also, have you enabled it correctly as written in [documentation ](https://github.com/jenssegers/laravel-mongodb#lumen)?
